app/api/canvas/service/canvas.service.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
+
+const async = require('async');
+
+const models = require('../../../models/'),
+    CanvasModel = models.Canvas,
+    UsersModel = models.Users;
+
+const SuccessHandler = require('../../../util/').SuccessHandler;
+
+
+module.exports = {
+
+    /**
+     * @type function
+     * @access public
+     * @param req
+     * @param res
+     * @param next
+     * @author Albert Hambardzumyan <hambardzumyan.albert@gmail.com>
+     * @description Link account.
+     */
+    linkAccount: (req, res, next) => {
+        const userId = req.body['userId'];
+        const token = req.body['token'];
+
+        async.waterfall([
+            (callback) => {
+                UsersModel.linkAccount(userId, token, callback);
+            }
+        ], (err, result) => {
+            if (err) next(err);
+            else SuccessHandler.handleAdd(res, next, result);
+        });
+    },
+
+    /**
+     * @type function
+     * @access public
+     * @param req
+     * @param res
+     * @param next
+     * @author Albert Hambardzumyan <hambardzumyan.albert@gmail.com>
+     * @description Import.
+     */
+    import: (req, res, next) => {
+        const token = req.body['token'];
```

Comment:
You can use object destructuring here instead of specifying token as string. `const {token} = req.body;`